site/releases/spark-release-2-4-2.html

```diff
@@ -205,10 +205,13 @@ <h2>Spark Release 2.4.2</h2>
 
 <p>Spark 2.4.2 is a maintenance release containing stability fixes. This release is based on the branch-2.4 maintenance branch of Spark. We strongly recommend all 2.4 users to upgrade to this stable release.</p>
 
+<p>Note that Scala 2.11 support was deprecated in 2.4.1, and as of 2.4.2, the pre-built convenenience binaries are compiled for Scala 2.12.
```

Comment:
Maybe 2.4.1+?

Reply:
You mean "deprecated as of 2.4.1 and beyond"? Ok that's fine.

Reply:
Yeah.